src/mame/taito/taito_f3.cpp

```diff
@@ -9,19 +9,10 @@
     Major thanks to Aaron Giles for sound info, figuring out the 68K/ES5505
     rom interface and ES5505 emulator!
     Thanks to Acho A. Tang for Kirameki Star Road sound banking info!
-    Thank you to Shiriru for the scanline rendering (including alpha blending),
```

Comment:
Let's keep it in. It's Bryan thanking Shiriru for past contributions.

Reply:
Legacy, can as well go in header driver file as `// thanks-to: Shiriru`, usage example: https://github.com/mamedev/mame/blob/ca7daca08f78d85deb5c0b1af974f208063265df/src/mame/nec/pc8801.cpp#L1-L3

Reply:
done, thanks for the guidance here.